### PR TITLE
Fixes #28138 - Show CV data in publish task output

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: app/lib/actions/katello/content_view/publish.rb [hunk 1]
```diff
@@ -82,6 +82,9 @@ def run
               attrs[:metadata] = metadata
             end
           end
+
+          output[:content_view_id] = input[:content_view_id]
+          input[:content_view_version_id] = input[:content_view_version_id]
```
Comment (evgeni) [inline app/lib/actions/katello/content_view/publish.rb]:
```suggestion
          output[:content_view_version_id] = input[:content_view_version_id]
```

Comment (jturel) [reply app/lib/actions/katello/content_view/publish.rb]: Lol

**Comment by jturel** [comment]
> In [7265be3#diff-620d80d1e16354da4a45be23cd269036](https://github.com/Katello/katello/commit/7265be37ca1de617ae8b7c040f2f48be20db45b6#diff-620d80d1e16354da4a45be23cd269036), we also deleted
> 
> ```
>             output[:composite_version_auto_published]
>             output[:composite_view_publish_failed]
>             output[:composite_auto_publish_task_id]
> ```
> 
> no idea if that data was used by anyone, though

This was intentional. Composite view publish is handled via our internal event queue so there's no task to refer to here!

File: app/lib/actions/katello/content_view/publish.rb (hunk 1 above)
Comment (jturel) [reply app/lib/actions/katello/content_view/publish.rb]: Fixed

**Comment by sseelam2** [comment]
This PR is working as expected. I can see content_view_id and content_view_version_id in the output section of the response. ACK :+1: @jturel

**Comment by jjeffers** [comment]
I'm attempting to test this, but I'm hitting an error in the publish task. 

```
/home/vagrant/projects/katello/app/models/katello/glue/pulp/repo.rb:25:in `block (2 levels) in included'
/home/vagrant/projects/katello/app/lib/katello/lazy_accessor.rb:118:in `instance_eval'
/home/vagrant/projects/katello/app/lib/katello/lazy_accessor.rb:118:in `run_initializer'
/home/vagrant/projects/katello/app/lib/katello/lazy_accessor.rb:97:in `lazy_attribute_get'
/home/vagrant/projects/katello/app/lib/katello/lazy_accessor.rb:44:in `block (2 levels) in lazy_accessor'
/home/vagrant/projects/katello/app/models/katello/glue/pulp/repo.rb:272:in `published?'
/home/vagrant/projects/katello/app/lib/actions/katello/repository/check_matching_content.rb:27:in `run'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/dynflow-1.3.0/lib/dynflow/action.rb:538:in `block (3 levels) in execute_run'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/dynflow-1.3.0/lib/dynflow/middleware/stack.rb:26:in `pass'
[...]
```

**Comment by jturel** [comment]
@jeremylenz can you try with a fresh repo and cv? not sure what to make of that error, but definitely seems unrelated to my change. just tested and it works for me locally as well with all the latest foreman and katello code underneath

**Comment by jjeffers** [comment]
> @jjeffers can you try with a fresh repo and cv? not sure what to make of that error, but definitely seems unrelated to my change.

I agree - it doesn't appear to be related. I'll retry ASAP.

**Pull request merged**